layouts/books/list.html

```
@@ -1,45 +1,46 @@
-    <ul class="book-posts">
-        {{ range $paginator.Pages.GroupByDate "2006-01-02" }} {{ range $index,
-        $page := .Pages }} {{ $url := urls.Parse .Params.original_url}}
+    {{ .Content }}
+
+    {{ if .Data.Singular }}
+    <h3 class="section-filter">{{ i18n "filtering-for" }} "{{ .Title }}"</h3>
+    {{ end }}
+
+    <div class="view-toggle">
+        <a class="active">List</a>
+        <a href="{{ "/media/books/grid" | relLangURL }}">Grid</a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)